src/negotiation/http/maker_client.ts

```diff
@@ -0,0 +1,25 @@
+/// An Http implementation of the taker negotiation service
+
+import axios from "axios";
+import { Order } from "../payload";
+import { MakerClient } from "../taker";
+
+export class HttpMakerClient implements MakerClient {
+  private readonly makerUrl: string;
+
+  constructor(makerUrl: string) {
+    this.makerUrl = makerUrl;
+  }
+
+  public async getOrder(tradingPair: string) {
+    const response = await axios.get(`${this.makerUrl}/${tradingPair}`);
+    return response.data;
+  }
+
+  public async getExecutionParams(order: Order) {
```

Comment:
This was carrying a terminology of `accept` in the example.

However, as this endpoint is readonly (`HTTP GET`) then it was confusing to have a `write` terminology of `accepting` an order.

If we were to actually tell the maker that the taker `accepts` the order, then I believe it should make a difference for the `maker`.

As we were not, I just changed that to `getExecutionParams` which is a `readonly` action (`get`).

Now, if the intent is for the maker to have an impact on the fact that a taker takes an offer, for example, by delisting the order as it has been accepted, then comment here and let me know.

In this case, I would change the terminology and actually make the maker delist its order (or marked it as accepted).

Reply:
Yep, thank you for clarifying.